src/modules/Navbar/Navbar.css

```diff
@@ -0,0 +1,89 @@
+.NavbarItems {
+    background: linear-gradient(90deg, rgb(0, 0, 0) 0%, rgb(6, 6, 7) 100%);
+    height: 100px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 1.2rem;
+}
+
+
+.navbar-logo{
+    color:white;
+    position:absolute;
+    left: 50%;
+    margin-left: -50px !important;  /* 50% of your logo width */
```

Comment:
!important is used to override all the other CSS "margin-left"s. I think you just want to leave the comment.

Reply:
Hardcoded pixels as well.

Reply:
Also, I would not recommend using !important for the future because I think it's not best practice.